test/install.mk

```diff
@@ -119,10 +120,15 @@ install-egress:
 # Telemetry will be installed in istio-control for the tests, until integration tests are changed
 # to expect telemetry in separate namespace
 install-telemetry:
-	#bin/iop istio-telemetry istio-grafana $IBASE/istio-telemetry/grafana/ 
+	bin/iop ${ISTIO_NS} istio-grafana ${BASE}/istio-telemetry/grafana/  ${IOP_OPTS}
 	bin/iop ${ISTIO_NS} istio-prometheus ${BASE}/istio-telemetry/prometheus/  ${IOP_OPTS}
-	bin/iop ${ISTIO_NS} istio-mixer ${BASE}/istio-telemetry/mixer-telemetry/  ${IOP_OPTS}
-	kubectl wait deployments istio-telemetry prometheus -n ${ISTIO_NS} --for=condition=available --timeout=${WAIT_TIMEOUT}
+	bin/iop ${ISTIO_NS} istio-telemetry ${BASE}/istio-telemetry/mixer-telemetry/  ${IOP_OPTS}
+
+	kubectl wait deployments istio-telemetry prometheus grafana -n ${ISTIO_NS} --for=condition=available --timeout=${WAIT_TIMEOUT}
+
+# Install kiali separately with telemetry
+install-kiali:
+	bin/iop ${ISTIO_NS} istio-kiali ${BASE}/istio-telemetry/kiali/ ${IOP_OPTS}
```

Comment:
We use common ISTIO_NS for other components because the tests have some hardcoded values.
Unless Kiali has some e2e tests that expect to find it in same ns with pilot - can you install it in istio-admin ?

Reply:
changed it to istio-admin